resources/user-agents/browsers/yabrowser/yabrowser-15-on.json

```diff
@@ -327,37 +329,61 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) AppleWebKit/* (*khtml*like*gecko*) YaBrowser/#MAJORVER#.#MINORVER#* */*",
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#) AppleWebKit/* (*khtml*like*gecko*)* YaBrowser/#MAJORVER#.#MINORVER#* Mobile/*",
```

Comment:
I restructured this a little bit because it doesn't seem that the iPhone ever uses the `iOS_C_*` platform patterns, at least not for this browser. So this should reduce the pattern count a little bit.